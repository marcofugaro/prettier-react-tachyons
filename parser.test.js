const prettier = require("prettier");
const parser = require("./parser");

test("reformatting examples", () => {
  expect(
    prettier.format(`<div className='h1 h1 w2 pv2 bg-transparent red' />`, {
      parser
    })
  ).toMatchSnapshot();
  expect(
    prettier.format(`<div className='unknown pv2 ph2' />`, {
      parser
    })
  ).toMatchSnapshot();
});
