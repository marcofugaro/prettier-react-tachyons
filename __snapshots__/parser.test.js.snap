// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reformatting examples 1`] = `
"<div className=\\"w2 h1 pv2 red bg-transparent\\" />;
"
`;

exports[`reformatting examples 2`] = `
"<div className=\\"ph2 pv2 unknown\\" />;
"
`;
